string actionName
string[] argNames
---
int64 success_bool
